suite/auto-sync/README.md

```diff
@@ -15,6 +15,9 @@ Please refer to [intro.md](intro.md) for an introduction about this tool.
 
 ## Install
 
+> ⚠️
+> Auto-Sync only runs on Linux, running on Windows is desirable but not a priority at the moment
```

Comment:
Better just state it and link the issue: https://github.com/capstone-engine/capstone/issues/2755.
So people have a place to check for updates and complain. Also below.